.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=1234
+MYSQLDB_DATABASE=book_store
+MYSQLDB_LOCAL_PORT=3308
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
how happened that you pushed your .env to GitHub 😭 Please remove it! this file contains  sensitive  information

Reply:
you can add `.env.sample` but it file without values (for now it’s an optional comment)